pkg/ccl/backupccl/schedule_exec.go

```diff
@@ -275,7 +275,11 @@ func (e *scheduledBackupExecutor) GetCreateScheduleStatement(
 	}
 
 	node := &tree.ScheduledBackup{
-		ScheduleLabel:   tree.NewDString(sj.ScheduleLabel()),
+		// ScheduleLabelSpec.IfNotExists is always set to false here as this attribute is only relevant
```

Comment:
hmm the comment is a little confusing. The reason why it is okay to default `IF NOT EXISTS` to false here is that this method powers `SHOW CREATE SCHEDULE` which is run against a schedule that is currently present in the cluster. Whether that schedule was created/or not created with IF NOT EXISTS seems irrelevant and not enough to convince me to stick a field in the job record. I'd say nuke the comment as it brings to attention something that is not necessarily important. If in the future someone complains about this discrepancy we know how to fix it :shrug.

Reply:
makes sense! done.